CoffeeChat/Controller/LoginViewController.swift

```diff
@@ -98,7 +98,7 @@ extension LoginViewController: GIDSignInDelegate, MessageAlertViewDelegate {
             return
         }
 
-        if let email = user.profile.email, !(email.contains("@cornell.edu")) {
+        if let email = user.profile.email, !(email.contains("@cornell.edu")) && email != "cornellpearapp@gmail.com" {
```

Comment:
I don't think you need parentheses around `email.contains(...)` after the bang. Also, why do we explicitly blacklist `cornellpearapp@gmail.com`? Is this documented somewhere? I think we should extract this into `Constants` or something since putting it in here raw is not great and is easy to forget if this address changes in the future.